src/components/Home/SubmissionDetailTable.js

```diff
@@ -0,0 +1,170 @@
+import KeyboardBackspaceIcon from "@material-ui/icons/KeyboardBackspace"
+
+const useStyles = makeStyles(theme => ({
+  backIcon: {
+    "&:hover": {
+      cursor: "pointer",
+    },
+  },
+  card: {
+    border: "none",
+    padding: theme.spacing(0),
+  },
+  cardHeader: {
+    fontSize: "0.5em",
+    padding: 0,
+    marginTop: theme.spacing(1),
+    "&:hover": {
+      cursor: "pointer",
+    },
+  },
+  tableHeader: {
+    padding: theme.spacing(1),
+    backgroundColor: theme.palette.primary.main,
+    color: theme.palette.primary.contrastText,
+  },
+  tableIcon: {
+    minWidth: 35,
+    minHeight: 35,
+    justifyContent: "center",
+    alignItems: "center",
+    backgroundColor: theme.palette.common.white,
+    marginRight: theme.spacing(1),
+  },
+  headRows: {
+    fontWeight: "bold",
+  },
+  objectType: {
+    "&:first-letter": {
+      textTransform: "capitalize",
+    },
+  },
+}))
+
+const headRows = ["Title", "Object type", "Status", "Last modified", "", "", "", ""]
+
+type ObjectDetails = {
+  accessionId: string,
+  lastModified: string,
+  objectType: string,
+  status: string,
+  title: string,
+}
+
+type SubmissionDetailTableProps = {
+  folderTitle: string,
+  bodyRows: Array<ObjectDetails>,
+  folderType: string,
+  onClickCardHeader: () => void,
+  onDelete: (objectId: string, objectType: string, objectStatus: string) => void,
+}
+
+const SubmissionDetailTable = (props: SubmissionDetailTableProps) => {
+  const classes = useStyles()
+  const { bodyRows, folderTitle, folderType, onClickCardHeader, onDelete } = props
+
+  const getDateFormat = (date: string) => {
+    const d = new Date(date)
+    const day = d.getDate()
+    const month = d.getMonth() + 1
+    const year = d.getFullYear()
+    return `${day}.${month}.${year}`
+  }
+
+  return (
+    <Card className={classes.card} variant="outlined">
+      <CardHeader
+        className={classes.cardHeader}
+        avatar={<KeyboardBackspaceIcon className={classes.backIcon} />}
+        title={`Your ${folderType} submissions`}
+        titleTypographyProps={{ variant: "subtitle1", fontWeight: "fontWeightBold" }}
+        onClick={onClickCardHeader}
+      />
+      <CardContent>
+        <TableContainer component={Paper}>
+          <Table>
+            <TableHead>
+              <TableRow>
+                <TableCell colSpan={8} padding="none">
+                  <ListItem dense className={classes.tableHeader}>
+                    <ListItemIcon className={classes.tableIcon}>
+                      {folderType === "published" ? <FolderIcon color="primary" /> : <FolderOpenIcon color="primary" />}
+                    </ListItemIcon>
+                    <ListItemText primary={folderTitle} />
+                    <Button color="secondary" disabled aria-label="Edit current folder">
+                      Edit
+                    </Button>
+                    <Button variant="contained" disabled aria-label="Publish current folder">
```

Comment:
I think this is the change necessary to disable buttons when folder published, but no 100% sure
We can keep the buttons here but add an issue about adding functionality to them and i think it is needed to be able to browse via some URL to these folders

```suggestion
                    <Button disabled={folderType === "published"} variant="contained"  aria-label="Publish current folder">
```